src/site/antora/modules/ROOT/examples/hibernate/log4j2.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to you under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<Configuration xmlns="https://logging.apache.org/xml/ns"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="
+                   https://logging.apache.org/xml/ns
+                   https://logging.apache.org/xml/ns/log4j-config-2.xsd">
+
+  <Appenders>
+    <Console name="CONSOLE">
+      <PatternLayout/>
+    </Console>
+  </Appenders>
+
+  <Loggers>
+
+    <!-- Log just the SQL -->
+    <Logger name="org.hibernate.SQL" level="DEBUG"/>
+
+    <!-- Log JDBC bind parameters and extracted values -->
+    <Logger name="org.hibernate.type" level="TRACE"/>
+    <Logger name="org.hibernate.orm.jdbc.bind" level="TRACE"/>
+    <Logger name="org.hibernate.orm.jdbc.extract" level="TRACE"/>
```

Comment:
I would keep this commented out with a warning:
> Be aware that JDBC statement parameters can contain sensitive data (passwords, credit card numbers, etc.).
> Uncomment this **only** in a dev environment (or start looking for a new job :wink:).
> See also "Arbiters" on how to switch config based on the environment.

Reply:
Fixed in 2d86aa07c5997d912e0c5dcc0243b0b79e9b967e.